src/libConsensus/ConsensusLeader.cpp

```diff
@@ -817,16 +818,27 @@ ConsensusLeader::ConsensusLeader(
   m_commitPointMap.at(m_myID) = *m_commitPoint;
   m_commitCounter = 1;
 
-  m_allCommitsReceived = false;
+  m_sufficientCommitsReceived = false;
   m_commitRedundantCounter = 0;
   m_commitFailureCounter = 0;
   m_numSubsetsRunning = 0;
 
+  m_txnProcessingTimeout = std::max(
+      0, ((int)MICROBLOCK_TIMEOUT -
+          ((int)TX_DISTRIBUTE_TIME_IN_MS + (int)FINALBLOCK_DELAY_IN_MS) / 1000 -
+          (int)CONSENSUS_OBJECT_TIMEOUT) /
+             2);
+
+  m_sufficientCommitsNumForSubsets =
+      ceil((double)m_committee.size() * COMMIT_TOLERANCE_PERCENT / 100);
+
   LOG_GENERAL(INFO, "Consensus ID      = " << m_consensusID);
   LOG_GENERAL(INFO, "Leader/My ID      = " << m_myID);
   LOG_GENERAL(INFO, "Committee size    = " << committee.size());
   LOG_GENERAL(INFO, "Num for consensus = " << m_numForConsensus);
   LOG_GENERAL(INFO, "Num for failure   = " << m_numForConsensusFailure);
+  LOG_GENERAL(INFO, "Num for commits before subset-creation = "
```

Comment:
Maybe we log this only for m_DS == true?

Reply:
ok